### ISSUE TITLE
Highstock navigator black in styled mode

### ISSUE THREAD

**Issue by 3Ue** [highcharts/highcharts#6339]
#### Expected behaviour
Render the navigator correctly in styled mode.

#### Actual behaviour
Renders the navigator outside-area black.

#### Live demo with steps to reproduce
http://jsfiddle.net/gh/get/jquery/1.7.2/highcharts/highcharts/tree/master/samples/highcharts/css/stock-navigator/

![navigator-black](https://cloud.githubusercontent.com/assets/5176160/22640896/d9b95be2-ec54-11e6-85c1-91694a560acf.PNG)

#### Affected browser(s)
Latest FF, IE and Chrome.

**Comment by pawelfus** [comment]
Thank you for reporting! CSS styles for `.highcharts-navigator-mask-outside` are missing, workaround:
```css
.highcharts-navigator-mask-outside {
  fill: transparent
  // or fill-opacity: 0
}
```

Demo: http://jsfiddle.net/hnsac3bL/1/

*Internal note:*
@TorsteinHonsi - the changes are in .scss file, but not in the released package.

**Comment by TorsteinHonsi** [comment]
It appears that the `highcharts.css` file hasn't been published since v5.0.2.

See http://code.highcharts.com/5.0.3/css/highcharts.css and http://code.highcharts.com/5.0.2/css/highcharts.css .

**Comment by sebastianbochan** [comment]
+1

**Comment by jon-a-nygaard** [comment]
_Internal Note: The gulp task styles returned before it actually was completed. Leading to copyToDist beeing executed before the css was created and saved. Fixed for the upcoming release 5.0.8.
Remaining: upload a build of highcharts.css to versions 5.0.3-5.0.7_

**Comment by jon-a-nygaard** [comment]
Uploaded missing style sheets for versions 5.0.3 to 5.0.7, and updated version 5 and 5.0 with latest stylesheet.
